examples/meta/src/preprocessor/random_fourier_features.sg.in

```diff
@@ -8,7 +8,8 @@ Features feats_test = create_features(data_test)
 #![create_features]
 
 #![create_preprocessor]
-Transformer preproc = create_transformer("RandomFourierGaussPreproc", seed=12345)
+Kernel gaussiankernel = create_kernel("GaussianKernel", log_width=-0.34657359027997264)
```

Comment:
"gaussiankernel" is a bad varname. Use "gaussian_kernel"  or yet better just "kernel" or even "k"